datalad/support/tests/test_annexrepo.py

```diff
@@ -1134,7 +1134,8 @@ def test_annex_copy_to(origin, clone):
     repo.get("test-annex.dat")
     # now it has:
     eq_(repo.copy_to("test-annex.dat", "target"), ["test-annex.dat"])
-    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), ["test-annex.dat"])
+    # and will not be copied again since it was already copied
+    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), [])
```

Comment:
Well, actually that's not exactly what I meant. (Although I'm fine with that). I'm concerned about a possible non-zero exit - yes. But this could happen by mistransforming a passed list for example.
Or just respecting the first entry or not existing files, etc.
Your change in that single test line means, that we no longer test, whether we can successfully pass a list, because the only assertion involving a list of files is this one and expects an empty result. So, theoretically the function could be unable to deal with a list at all and simply always return an empty result when a list was passed. We wouldn't notice now. So, what I actually wanted to see is just an invocation, that uses a list and expects something in the results. ;-)

And since the `_run_annex_json` behaves differently I'm not sure how it now behaves on not existing files or files not in annex, etc. Before we just got a non-zero exit from annex and the respective `CommandError`. We didn't need to test this explicitly, we simply relied on git-annex' non-zero exit. Now, we have no clue what happens in such cases and I think, git-annex itself behaves differently with `--json`. So actually we need to test for it now. Your mockup doesn't do that.

So just two test lines, please:
 - pass a list, that should result in something else than doing nothing
 - try to copy a not existing file
A file not in annex we already tried to copy (`INFO.txt`), so that's fine, if we have make sure that the list call works at all.